Sources/BraveWallet/Panels/Signature Request/SignInWithEthereumView.swift

```diff
@@ -0,0 +1,243 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveStrings
+import BraveCore
+import DesignSystem
+
+/// View for showing `SignMessageRequest` for  ethSiweData
+struct SignInWithEthereumView: View {
+  
+  let account: BraveWallet.AccountInfo
+  let originInfo: BraveWallet.OriginInfo
+  let message: BraveWallet.SIWEMessage
+  var action: (_ approved: Bool) -> Void
+  
+  @State private var isShowingDetails: Bool = false
+  @Environment(\.sizeCategory) private var sizeCategory
+  
+  var body: some View {
+    ScrollView {
+      VStack(spacing: 10) {
+        faviconAndOrigin
+        
+        messageContainer
+        
+        buttonsContainer
+          .padding(.top)
+          .opacity(sizeCategory.isAccessibilityCategory ? 0 : 1)
+          .accessibility(hidden: sizeCategory.isAccessibilityCategory)
+      }
+      .padding()
+    }
+    .overlay(
+      Group {
+        if sizeCategory.isAccessibilityCategory {
```

Comment:
I assume this is copied from somewhere else but fwiw: You can swap to the new version of `overlay` that takes a `ViewBuilder` so you wouldn't need the `Group`:

```swift
.overlay(alignment: .bottom) {
  if sizeCategory.isAccessibilityCategory { ... }
}
```